app/controllers/test/base_controller.rb

```diff
@@ -1,4 +1,6 @@
 class Test::BaseController < ApplicationController
   skip_before_action :verify_authenticity_token
   skip_before_action :authenticate_user!
+
+  disable_site_header!
```

Comment:
There was a problem with the "Run tests" button not being  seen in the test layout because the header was present. I disabled it for now, and will fix the CSS in another PR.